files/conf.d/log4pot.yml

```diff
@@ -0,0 +1,14 @@
+---
+- type: log
+  paths:
+    - /data/log4pot/log/log4pot.log
+  scan_frequency: 5s
+  fields_under_root: true
+  encoding: utf-8
+  ignore_older: 3h
+  tags: ["log4pot"]
+  fields:
+    t-pot_ip_ext: ${MY_EXTIP}
+    t-pot_ip_int: ${MY_INTIP}
+    t-pot_hostname: ${MY_HOSTNAME}
+    type: log4pot
```

Comment:
Can you make this `Log4pot`? Don't know if elastic is case-sensitive, but that is the way it is in the logstash config.

Reply:
updated :)